examples/fit_fake_data.py

```diff
@@ -47,22 +46,20 @@
 # }}}
 #}}}
 #{{{lmfitdata method
-thisfit = lmfitdata(empty_data)
 newfit = lmfitdata(fake_data)
 newfit.functional_form = (Mi + (M0-Mi)*sp.exp(-(vd*R1)))
-thisfit.functional_form = newfit.functional_form
 newfit.set_guess(
-        M0 = dict(value=-500, max=0, min=-501),
-        Mi=dict(value=500, max = 501, min=0), 
-        R1=dict(value=5, max = 6, min = 1))
+        M_0 = dict(value=-500, max=0, min=-501),
+        M_inf=dict(value=500, max = 501, min=0), 
+        R_1=dict(value=5, max = 6, min = 1))
```

Comment:
it knows the names by their symbols, not by the variable names you've assigned those symbols to.